lib/foods.js

```diff
@@ -15,7 +15,7 @@ function checkInputs() {
     var food = $('#name-field input').val();
     var calories = $('#calories-field input').val();
 
-    $('tbody').append('<tr class="food-row"><td>' + food + '</td><td class="food-calories">' + calories + '</td><td class="food-delete"><button>-</button></td></tr>');
+    $('tbody').prepend('<tr class="food-row"><td>' + food + '</td><td class="food-calories">' + calories + '</td><td class="food-delete"><button>-</button></td></tr>');
```

Comment:
@lucyconklin I like the update to `prepend` which will place the new food item at the top of the table instead of the bottom as it did with the `append` instead.